docs/releaseNotes.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Apache DistributedLog 0.5.0 Release Notes
+---
+
+This is the second Apache release of DistributedLog and the first release as a sub-project of Apache BookKeeper!
+
+The 0.5.0 release upgrades BookKeeper from Twitter `4.3.7-TWTTR-OSS` version to the official Apache `4.5.0` version.
+It is a big milestone in Apache BookKeeper & DistributedLog community, converaging the developement efforts for
+both communities and moving the distributedlog developement on the official Apache bookkeeper release.
+
+Apache DistributedLog users are encouraged to upgrade to 0.5.0. The technical details of this release are summarized
+below.
+
+## Highlights
+
+### BookKeeper Verion Upgrade
+
+The main change in 0.5.0 is bookkeeper version upgrade. We upgraded the bookkeeper version from Twitter's `4.3.7-TWTTR-OSS`
+to the official Apache `4.5.0` version. This upgrade brings in bookkeeper security feature and performance improvement (such as netty 4 upgrade).
```

Comment:
add a link to bookkeeper 4.5.0 release notes.

Reply:
Thanks. will add it.

Reply:
Thanks. will add it.